include/AutomationEditor.h

```diff
@@ -241,6 +241,8 @@ protected slots:
 	QScrollBar * m_leftRightScroll;
 	QScrollBar * m_topBottomScroll;
 
+	void incrementLeftRightScoll(int value);
```

Comment:
I have just noticed that something like `update` or `adjust` would have been better than `increment`. The reason is that it does not always increment but can also decrement depending on the sign of `value`.

However, I don't want to be too picky. :sweat_smile:

Reply:
No problem!